src/msg/simple/Pipe.cc

```diff
@@ -466,6 +466,16 @@ int Pipe::accept()
     existing = msgr->_lookup_pipe(peer_addr);
     if (existing) {
       existing->pipe_lock.Lock(true);  // skip lockdep check (we are locking a second Pipe here)
+      if (existing->state == STATE_CLOSED) {
+        ldout(msgr->cct, 1) << __func__ << " existing racing replace or mark_down happened while replacing."
+                            << " existing_state=" << existing->get_state_name() << dendl;
+	reply.tag = CEPH_MSGR_TAG_RETRY_GLOBAL;
```

Comment:
https://github.com/ceph/ceph/commit/a640ca93307e92f8032881e35fdd74b60f0091d0

retry_session will cause side effect we may not want to have